File: GestionMorgueFinal/src/app/@core/backend/common/model/CertificatTransfertCorps.ts
import {Medecins} from './Medecins';
import {Decedes} from './Decedes';

export class CertificatTransfertCorps {
id: number;
medecins: Medecins ;
cercueilType: string;
declaration: Date;
remarque: string ;
defunt: Decedes ;
declarant: string;
tel: string;
destination: string ;
mortuaire: string;
inhumationSociete: string;
cin: string;
  constructor() {
  }

}

File: GestionMorgueBack/src/main/java/com/akveo/bundlejava/model/CertificatTransfertCorps.java
package com.akveo.bundlejava.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.OneToOne;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.GenerationType;
import java.util.Date;
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class CertificatTransfertCorps {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    @OneToOne
    private Medecins medecins;
    private String cercueilType;
    private Date declaration;
    private String remarque;
    @OneToOne
    private Decedes defunt;
    private String declarant;
    private String tel;
    private String destination;
    private String mortuaire;
    private String inhumationSociete;
    private String cin;

}
